source/overview/license-and-subscription.rst

```diff
@@ -109,6 +109,71 @@ Is my license available immediately?
+
+
+What happens to my trial/subscription if I don't renew in time?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+From version 5.24, if you don't renew within the 60-day renewal period, a 10 day grace period is provided. During this period your Mattermost installation runs as normal, with full access to Enterprise features. 
+
+When the grace period expires, your Enterprise version is downgraded to Team Edition. Enterprise features are disabled.
+ 
+What happens when my trial/subscription expires?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you don't renew your license within the 10-day grace period, your Mattermost version is automatically downgraded to Team Edition so you can still access and use Mattermost. However, Enterprise features will no longer be available and if you are currently using them, the functionality will no longer be accessible. 
+
+When you renew, the Enterprise features will become available with the previous configuration (provided no action such as user migration has been taken). 
+
+Which features are affected when my Enterprise license expires?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you are using the following Enterprise features, they will be impacted 
+
+.. csv-table::
+    :header: "Feature", "How it's affected", "What steps do I need to take?"
+
+    "Elasticsearch", "Elasticsearch is automatically disabled and will start using the default database for indexing posts", "None needed."
+    "AD/LDAP, SAML SSO, Office 365 SSO, and Google SSO login options removed from the sign-in page.", "- Users who previously signed in with one of these methods are no longer able to. 
+    - Users who were already signed in can continue to use Mattermost until their session expires or until they log out.", "Users must be migrated to email authentication using the `CLI <https://docs.mattermost.com/administration/command-line-tools.html#mattermost-user-migrate-auth>`_. This can also be done via **System Console > Users**. Select the drop-down menu for the relevant member, choose **Switch to Email/Password**, enter a new password, and choose **Reset**."
+    "AD/LDAP groups in the database are retained but cannot be used", "- Memberships are frozen in state for group synced teams/channels. 
+    - Mentions for AD/LDAP groups are not shown in the autocomplete menu. 
+    - Group mentions are no longer highlighted in text and do not trigger new notifications.", "Use the `CLI <https://docs.mattermost.com/administration/command-line-tools.html#mattermost-group>`_ to modify group sync settings for the team/channel."
+    "High availability is disabled.", "If all nodes in a cluster continue running, the nodes will stop communicating and caches will get out of sync. This is likely to cause delays in messages, notifications, etc.", "None needed."
```

Comment:
"If all nodes in a cluster continue running..." --> does this indicate the admin does need to take action so that the nodes are no longer been used?

Reply:
I couldn't find anything in [this guide](https://docs.mattermost.com/deployment/cluster.html) about shutting down/discontinuing a cluster other than for maintenance. I'm not certain which of the steps would apply in this instance.

Reply:
@justinegeffen we could check in with the Enterprise team to confirm

Reply:
Asked them here: https://community.mattermost.com/core/pl/j9rza84tipy1tk7yc3bn1ho8qy.